cmd/http-stats.go

```diff
@@ -143,6 +143,7 @@ type HTTPStats struct {
 	rejectedRequestsHeader  uint64
 	rejectedRequestsInvalid uint64
 	s3RequestsInQueue       int32
+	incomingS3Requests      uint64
```

Comment:
```suggestion
	s3RequestsInComing      uint64
```

Reply:
With same format style as `s3RequestsInQueue`

Reply:
> s3RequestsInComing

I did this, s3RequestsIncoming